analyze/edge-light.ts

```diff
@@ -1,4 +1,10 @@
-import type { ArcjetLogger, ArcjetRequestDetails } from "@arcjet/protocol";
+import type {
+  ArcjetLogger,
+  ArcjetRequestDetails,
+  CustomDetect,
+  DetectSensitiveInfoResult,
+  SensitiveInfoConfig,
```

Comment:
Shouldn't pull types from protocol. Analyze is mostly a standalone API that we call into. We probably shouldn't even rely on the ArcjetLogger or ArcjetRequestDetails types.